ipfs-cids/.moonriver-cid

```diff
@@ -0,0 +1 @@
+QmRbAMAL5xztdhDcQhAeJhmsipa75sbPpQmbSKV5bAimHK
```

Comment:
why is it different from https://github.com/nova-wallet/subquery-nova/pull/174/files#diff-fff0807326c5169bd153ae07855d18ce8ca876877eb09b27c1bb9f020eccc5aa ?

Reply:
After each updates that id will change, as files inside IPFS container was changed too